subsys/bluetooth/mesh/Kconfig

```diff
@@ -733,7 +733,7 @@ config BT_MESH_LPN_INIT_POLL_TIMEOUT
 config BT_MESH_LPN_SCAN_LATENCY
 	int "Latency for enabling scanning"
 	range 0 50
-	default 10
+	default 15
```

Comment:
I do not think this is good idea to have default one value but to test another one. 
I didn't check it on the real device since it doesn't have a sense. It will variate on chip, frequency, controller etc.
If extended advertiser is used the current default value is boundary for poll reply (and depends even on the command). Some of replays happen before the scanner is on. Legacy advertiser is slow enough for enabling scanner with low latency value.
IMHO default value should allow stable test behavior. Each customer can tune it according his\her platform performance.